Node/Managing Node-NPM versions.md

```diff
@@ -0,0 +1,127 @@
+## Node version management
+
+### Node release schedule
+
+Node.js has defined and predictable release plan. Each node version goes through 3 phases: current, active(LTS) and maintenance. Be aware that odd-numbered versions will not go through active and maintenance phases.
+
+#### Current phase
+
+Incorporates most non-major changes that end up on `nodejs/node` main branch. New major releases are branched out every six months. Even-numbered versions are scheduled for release in April, while odd ones are scheduled for October.
+
+#### Active Long Term Support phase
+
+Versions in this phase include new features, bug fixes and updates that have been audited and determined to be stable for the release. With each odd-numbered release, previous even-numbered release will transition to LTS phase. Versions in this phase will receive active support for 12 months after they entered LTS phase.
+
+**You should always aim to use latest LTS version available on your project**
+
+#### Maintenance phase
+
+Versions in this phase will receive critical bug fixes and security updates for 18 months after it enters this phase.
+
+### Ensuring same version of Node is being used on all environments
+
+It is very important to keep the same versions of Node across the environments. When you update the version of Node in your project, you should take appropriate actions that the new version is also propagated/updated on build/production servers by notifying people responsible for those environments. By not running the same versions you could run into unexpected problems with failing builds, incorrect dependencies etc.
+
+### Managing versions
+
+There are couple of ways for managing versions of Node. I will list couple of options and generally, all options are fine as long as you are proficient at installing, updating and switching Node versions using that method. These probably aren't all available version managers on the internet, but are probably most popular.
+
+#### nvm - Node version manager
+
+[Node version manager](https://github.com/nvm-sh/nvm) is both powerful and easy to use and should be preferred method for managing Node versions.
```

Comment:
We talked on the leads meeting the other day and the conclusion is that nvm is not an ideal solution because it doesn't use the more standardised `.node-version` file.